lib/ts/recipe/emailpassword/types.ts

```diff
@@ -35,11 +35,33 @@ import {
 } from "../authRecipeModule/types";
 import RecipeImplementation from "./recipeImplementation";
 
+import { ComponentOverride } from "../componentOverride";
+import EmailPasswordSignInHeader from "./components/themes/signInAndUp/signInHeader";
+import EmailPasswordSignIn from "./components/themes/signInAndUp/signIn";
+import EmailPasswordSignInFooter from "./components/themes/signInAndUp/signInFooter";
+import EmailPasswordSignInForm from "./components/themes/signInAndUp/signInForm";
+import EmailPasswordSignUp from "./components/themes/signInAndUp/signUp";
+import EmailPasswordSignUpFooter from "./components/themes/signInAndUp/signUpFooter";
+import EmailPasswordSignUpForm from "./components/themes/signInAndUp/signUpForm";
+import EmailPasswordSignUpHeader from "./components/themes/signInAndUp/signUpHeader";
+
+export type ComponentOverrideMap = Partial<{
```

Comment:
Can we use "?" instead of `Partial`? Let's stick to one style of coding.

Reply:
Didn't know about any standard style across the repo, I'll change it.